ocs_ci/utility/utils.py

```
@@ -834,15 +834,17 @@ def exec_cmd(
-        )
+        run_kw = {
+            "stdout": subprocess.PIPE,
+            "stderr": subprocess.PIPE,
+            "timeout": timeout,
+            "env": _env,
+        }
+        # subprocess.run forbids stdin= and input= together; when callers pass input,
+        # stdin is managed internally.
+        if "input" not in kwargs:
+            run_kw["stdin"] = subprocess.PIPE
+        completed_process = subprocess.run(cmd, **run_kw, **kwargs)
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)